en/docs/getting-started/quick-start-guide/quick-start-guide-binary.md

```diff
@@ -27,7 +27,7 @@ Make sure to install and set up all the [installation prerequisites]({{base_pat
       ```
       
       ``` bash tab="Example"
-      micro-gw init petstore -a https://petstore.swagger.io/v2/swagger.json
+      micro-gw init petstore -a https://raw.githubusercontent.com/wso2/product-microgateway/master/samples/petstore_v3.yaml
```

Comment:
Some doc pages refer to https://github.com/wso2/product-microgateway/blob/master/samples/petstore_basic.yaml for the example. Just checking whether we have to change them as well

Reply:
Yes we need to change the swagger file of this sample only to use https endpoint url, as petstore does not support http endpoint anymore